src/test/java/net/starlark/java/eval/Benchmarks.java

```diff
@@ -185,19 +205,19 @@ public static void main(String[] args) throws Exception {
       for (Map.Entry<String, StarlarkFunction> e : benchmarks.entrySet()) {
         String name = e.getKey();
         System.out.flush(); // help user identify a slow benchmark
-        Benchmark b = run(name, e.getValue(), budgetNanos);
+        Benchmark b = run(name, e.getValue(), budgetNanos, iterations);
         if (b == null) {
           ok = false;
           continue;
         }
         System.out.printf(
-            "%-20s %10d %10s %10s %10d %10s\n",
+            "%-20s %10d %10s %10s %10s %10s\n",
             name,
             b.count,
-            formatDuration(((double) b.time) / b.count),
-            formatDuration(((double) b.cpu) / b.count),
-            b.steps / b.count,
-            formatBytes(b.alloc / b.count));
+            b.count != 0 ? formatDuration(((double) b.time) / b.count) : "?",
```

Comment:
Sorry, I should have foreseen that the zero case would be problematic. Let's reject iterations=0, but continue to use -1 as the sentinel value meaning "unset".

Reply:
Ping... once this is addressed I'll import the PR.

Reply:
Done.